src/App.tsx

```diff
@@ -1,17 +1,30 @@
 import "./App.css";
+import Header from "./components/Header";
+import { useAppSelector } from "../src/app/hooks"
```

Comment:
import for Header component is not necessary

Reply:
I tried to removed but it does not work